package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.1.7",
```

Comment:
Can you remove the version bump please? We'll take care of it when publishing ;)